pages/index.tsx

```diff
@@ -1,19 +1,41 @@
 import Head from 'next/head';
 import { PageWithLayout } from '@/lib/types/page';
-import { ReactElement } from 'react';
+import { ReactElement, useEffect, useState } from 'react';
 import Layout from '@/lib/components/Layouts/Layout';
 import { useSession, signOut } from 'next-auth/react';
+import ProductCard, { Product } from '@/lib/components/Cards/ProductCard';
 
 const Home: PageWithLayout = () => {
   const { data: session, status } = useSession();
+  const [products, setProducts] = useState([]);
+
+  useEffect(() => {
```

Comment:
For now, as you are just using a mock API move this `useEffect` and `products.map` to a new component, something like `Products.tsx`.

Reply:
Moved component